test/integration/functional_test.go

```diff
@@ -206,6 +210,38 @@ func validateStartWithProxy(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// validateSoftStart validates that after minikube already started, a "minikube start" should not change the configs.
+func validateSoftStart(ctx context.Context, t *testing.T, profile string) {
+	apiPortBefore := 8441
```

Comment:
For clarity, can you make this variable shared between the two tests, preferably with a verbose name?

Reply:
the handler func signature didnt allow a paramter. but I added it as an global var.